Dalamud/Game/Command/CommandManager.cs

```diff
@@ -39,6 +40,7 @@ internal CommandManager()
                 ClientLanguage.English => this.commandRegexEn,
                 ClientLanguage.German => this.commandRegexDe,
                 ClientLanguage.French => this.commandRegexFr,
+                ClientLanguage.ChineseSimplified => this.commandRegexCn,
```

Comment:
No `ClientLanguage.ChineseSimplified`

Reply:
Yeah, this won't work. You're going to have to check them for all languages.